### PR TITLE
Wemo insight: additional parameters added #3538

### PR DESCRIPTION
**Pull request by toby200**
Fix for #3538 -- added additional parameters for wemo insight plug

More info in the [forum thread] (https://community.openhab.org/t/wemo-insight-plug-adding-new-params-to-binding/4969) with @hmerk

### CODE REVIEW

**Comment by teichsta** [comment]
could you have a look into this PR please @hmerk?

**Comment by hmerk** [comment]
@teichsta, jes, for shure.
Will review end of the week, cause I'm on a business trip actually.

Best
Hans-Jörg

**Comment by teichsta** [comment]
perfect, thanks!

**Comment by hmerk** [comment]
@toby200, thanks for your contribution. Could you also update the WIKI with your addition, marked as available with version 1.8.

@teichsta LGTM, please feel free to merge for 1.8 release.

Best
Hans-Jörg

**Comment by toby200** [comment]
Thanks, [wiki page](https://github.com/openhab/openhab/wiki/Wemo-Binding) is updated.

Will likely be next week before I get openhab2 version done as I'm away a few days and I couldn't work out how to build the jars yet.

**Comment by hmerk** [comment]
@toby200 
Just found the last unknown parameter : It's the standBy Limit value
Could you add this as well, as I already prepared for ESH Binding.

Thanks
Hans-Jörg

**Comment by toby200** [comment]
Great - will look at this over the weekend.

**Comment by teichsta** [comment]
ok, so i'll wait for this addition before merging this PR, right?

**Comment by toby200** [comment]
Yes please

**Comment by toby200** [comment]
@hmerk @teichsta 
standbyLimit param is now added too. I've also updated the README.md

Thanks,
Toby

**Comment by hmerk** [comment]
@toby200 great, thanks!
@teichsta  LGTM

Best
Hans-Jörg

**Comment by toby200** [comment]
I neglected to set a rounding mode on the BigDecimal.divide() calls, which causes issues with certain specific values - will fix shortly so please hold off on the merge still

File: bundles/binding/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/WemoBinding.java [hunk 1]
```diff
@@ -196,6 +270,7 @@ protected void execute() {
 
 				} catch (Exception e) {
 					e.printStackTrace();
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/WemoBinding.java]: remove printStackTrace

**Comment by teichsta** [comment]
Thanks for incorporating the feedback @toby200 and sorry for stumbling about on last thing (remove printStackTrace()). Could you please remove this line? Thanks, Thomas E.-E.

**Comment by toby200** [comment]
No problem at all - I did wonder about it myself but wasn't sure and was hesitant to remove someone else's code. I've updated and squashed the commit, hope it looks ok now.

Many thanks, 
Toby

**Comment by teichsta** [comment]
great, thanks! Could you or @hmerk please finally update the wiki page accordingly?

**Comment by toby200** [comment]
All done :)

**Comment by teichsta** [comment]
great thanks! Looking forward to many more contributions from you, sir :+1:

**Pull request merged**